app/forms/nomination_request_form.rb

```diff
@@ -36,7 +36,7 @@ def save!
       ActiveRecord::Base.transaction do
         raise TooManyEmailsError if reached_email_limit
 
-        invite_schools_service.run([school.urn])
+        invite_schools_service.perform([school.urn])
```

Comment:
are you sure this change makes it async? from https://www.rubydoc.info/github/mperham/sidekiq/Sidekiq/Worker
should it not be `perform_async` on the class?

Reply:
@asmega No, it doesn't but I intentionally kept this one synchronous so that we can keep the transaction above it. I only wanted to make it async so that I can run this in the background when I send out emails to many 1000s of schools manually.